go/billing/models.py

```diff
@@ -176,6 +176,14 @@ class Meta:
         max_digits=10, decimal_places=3, default=Decimal('0.0'),
         help_text=_("The base cost per session in cents."))
 
+    session_unit_time = models.DecimalField(
+        max_digits=10, decimal_places=3, default=Decimal('20.0'),
+        help_text=_("The time of one billed unit (in seconds)."))
```

Comment:
@justinvdm My preference was to have a unit time specified in the message
cost (what's currently in the PR).